### ISSUE TITLE
ensure screenreaders can see headers on tables

### ISSUE THREAD

**Issue by brainwane** [OpenTechStrategies/psm#513]
For accessibility reasons, we should double-check that we comply with Section 508, clause G, Table Headers, when we use tables like `practiceLookupTableTemplate` and `tableLicenseTemplate`:

> If this is a data table, ensure row and column headers are identified using `th` elements.

**Comment by brainwane** [comment]
This problem has a few subtypes, I think, now that I'm using `pa11y` to look using the WCAG Level AA standard:
* `WCAG2AA.Principle1.Guideline1_3.1_3_1.H39.3.NoCaption`: If this table is a data table, consider using a caption element to the table element to identify this table. Example: `<table class=\"generalTable tablesorter\" id=\"draftTable\" cellspacing=\"0\" cellpadding=\"0\">\n                            <c...</table>`
* `WCAG2AA.Principle1.Guideline1_3.1_3_1.H42`: Heading markup should be used if this content is intended as a heading. Example: `<p><strong id=\"practiceLookupMatch...</p>`
* `WCAG2AA.Principle1.Guideline1_3.1_3_1.H73.3.NoSummary`: If this table is a data table, consider using the summary attribute of the table element to give an overview of this table. Example: `<table class=\"generalTable\">\n                    <thead>\n  ...</table>`
